### PR TITLE
React rails

### PR DESCRIPTION
**Pull request by NeckerFree**
**Project requirements**

- Create a new Rails app called 'hello-rails-react'.
- Use the commend rails new -j webpack my_new_rails_app as described in the previous lesson's tutorial.
- Verify that the jsbundling-rails gem is added to your Gemfile.
- Initialize your project with Git.
- Install needed node dependencies as described in the previous lesson's tutorial.
- Create a babel.config.js in the root folder of your project as described in the previous lesson's tutorial.
- Update webpack.config.js to include the babel loader as described in the previous lesson's tutorial.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
- Create a static view that will be the root of your app.

API:
API

APP:
App

### CODE REVIEW

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Elio, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that cannot be installed properly as `package.json` is missing in your project directory. Kindly to upload `package.json` at part of the project files to your Repository.

Also Linters are not showing on the Pull Request. Kindly ensure that Github actions is properly configured Linters

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,50 @@
+source 'https://rubygems.org'
```
Comment (Lordkaito) [inline Gemfile:1]: - [ ] Please make sure to add the `jsbundling-rails` gem as mentioned in the requirements for this project.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "devDependencies": {
```
Comment (Lordkaito) [inline package.json:2]: - [ ] Also, as mentioned in the requirements, you need to install `react-dom` and `babel-loader` in order to make the react side work as well, so please make sure to work on this issue.
- [ ] Also, make sure to setup `babel.config.js` and `webpack.config.js` as indicated [here](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a)

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @NeckerFree, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @NeckerFree  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the requested changes and installing the missing gems and dependencies

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**